Thehive project json fix (#18774)
demisto/content: Packs/TheHiveProject/ReleaseNotes/1_1_10.md

#### Integrations
##### TheHive Project

> - Fixed an issue where **thehive-create-case** and **thehive-update-case** would not parse **tags** field correctly

Reviewer: ```suggestion
- Fixed an issue in ***thehive-create-case*** and ***thehive-update-case*** commands where the *tags* argument was not parsed correctly.
```
